tests/vm_generator.go

```diff
@@ -0,0 +1,91 @@
+package tests
+
+import (
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1 "kubevirt.io/api/core/v1"
+)
+
+const virtio = "virtio"
```

Comment:
nit: const VirtIO
(this is like we are doing it on kubevirt)

Reply:
done